src/main/java/com/johanekstroem/parking/Controller/ParkingEventController.java

```diff
@@ -86,10 +85,11 @@ public ResponseEntity<ParkingEvent> startParking(@ModelAttribute("parkingevent")
                 .fromCurrentRequest()
                 .buildAndExpand(saveParkingEvent.getId())
                 .toUri();
-                return ResponseEntity.created(location).body(saveParkingEvent);
+                return "redirect:/saved";
               }
     }
-    return ResponseEntity.badRequest().build();
+
+    return "redirect:/ops";
```

Comment:
I was thinking if there is no parkingSpotId for => Long parkingSpotID = parkingEvent.getParkingSpot().getId();
nor car Id for =>   Long carID = parkingEvent.getCar().getId();
Then both redirect to /ops page would be also good!

Anyway wrong time redirect to ops page works and it looks neat! 👍